Multi-module: Failed to execute goal dev.hilla:hilla-maven-plugin:1.2.0.rc1:generate Index 1 out of bounds for length 0

### Description of the bug

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  4.385 s
[INFO] Finished at: 2022-09-04T12:20:29+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal dev.hilla:hilla-maven-plugin:1.2.0.rc1:generate (default-cli) on project enduser-vaadin: Execution default-cli of goal dev.hilla:hilla-maven-plugin:1.2.0.rc1:generate failed: Index 1 out of bounds for length 0 -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException
2022-09-04 10:20:30.039 ERROR 71148 --- [io-4886-exec-10] c.v.flow.server.DefaultErrorHandler      : 

java.lang.IllegalStateException: com.vaadin.flow.server.ExecutionFailedException: Hilla Generator execution failed with exit code 1
	at com.vaadin.base.devserver.AbstractDevServerRunner.getCause(AbstractDevServerRunner.java:768) ~[vaadin-dev-server-23.2.0.rc2.jar:na]
	at com.vaadin.base.devserver.AbstractDevServerRunner.getCause(AbstractDevServerRunner.java:764) ~[vaadin-dev-server-23.2.0.rc2.jar:na]
	at com.vaadin.base.devserver.AbstractDevServerRunner.handleRequest(AbstractDevServerRunner.java:627) ~[vaadin-dev-server-23.2.0.rc2.jar:na]
	at com.vaadin.flow.server.VaadinService.handleRequest(VaadinService.java:1564) ~[flow-server-23.2.0.rc2.jar:23.2.0.rc2]
	at com.vaadin.flow.server.VaadinServlet.service(VaadinServlet.java:364) ~[flow-server-23.2.0.rc2.jar:23.2.0.rc2]
	at com.vaadin.flow.spring.SpringServlet.service(SpringServlet.java:106) ~[vaadin-spring-23.2.0.rc2.jar:na]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:764) ~[tomcat-embed-core-9.0.60.jar:4.0.FR]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:227) ~[tomcat-embed-core-9.0.60.jar:9.0.60]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162) ~[tomcat-embed-core-9.0.60.jar:9.0.60]
	at org.apache.catalina.core.ApplicationDispatcher.invoke(ApplicationDispatcher.java:711) ~[tomcat-embed-core-9.0.60.jar:9.0.60]
	at org.apache.catalina.core.ApplicationDispatcher.processRequest(ApplicationDispatcher.java:459) ~[tomcat-embed-core-9.0.60.jar:9.0.60]
	at org.apache.catalina.core.ApplicationDispatcher.doForward(ApplicationDispatcher.java:353) ~[tomcat-embed-core-9.0.60.jar:9.0.60]
	at org.apache.catalina.core.ApplicationDispatcher.forward(ApplicationDispatcher.java:313) ~[tomcat-embed-core-9.0.60.jar:9.0.60]
	at org.springframework.web.servlet.mvc.ServletForwardingController.handleRequestInternal(ServletForwardingController.java:141) ~[spring-webmvc-5.3.18.jar:5.3.18]
	at org.springframework.web.servlet.mvc.AbstractController.handleRequest(AbstractController.java:177) ~[spring-webmvc-5.3.18.jar:5.3.18]
	at org.springframework.web.servlet.mvc.SimpleControllerHandlerAdapter.handle(SimpleControllerHandlerAdapter.java:51) ~[spring-webmvc-5.3.18.jar:5.3.18]
	at org.springframework.web.servlet.DispatcherServlet.doDispatch(DispatcherServlet.java:1067) ~[spring-webmvc-5.3.18.jar:5.3.18]
	at org.springframework.web.servlet.DispatcherServlet.doService(DispatcherServlet.java:963) ~[spring-webmvc-5.3.18.jar:5.3.18]
	at org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:1006) ~[spring-webmvc-5.3.18.jar:5.3.18]
	at org.springframework.web.servlet.FrameworkServlet.doGet(FrameworkServlet.java:898) ~[spring-webmvc-5.3.18.jar:5.3.18]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:655) ~[tomcat-embed-core-9.0.60.jar:4.0.FR]
	at org.springframework.web.servlet.FrameworkServlet.service(FrameworkServlet.java:883) ~[spring-webmvc-5.3.18.jar:5.3.18]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:764) ~[tomcat-embed-core-9.0.60.jar:4.0.FR]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:227) ~[tomcat-embed-core-9.0.60.jar:9.0.60]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162) ~[tomcat-embed-core-9.0.60.jar:9.0.60]
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53) ~[tomcat-embed-websocket-9.0.60.jar:9.0.60]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189) ~[tomcat-embed-core-9.0.60.jar:9.0.60]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162) ~[tomcat-embed-core-9.0.60.jar:9.0.60]
	at org.springframework.web.filter.RequestContextFilter.doFilterInternal(RequestContextFilter.java:100) ~[spring-web-5.3.18.jar:5.3.18]
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:117) ~[spring-web-5.3.18.jar:5.3.18]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189) ~[tomcat-embed-core-9.0.60.jar:9.0.60]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162) ~[tomcat-embed-core-9.0.60.jar:9.0.60]
	at org.springframework.web.filter.FormContentFilter.doFilterInternal(FormContentFilter.java:93) ~[spring-web-5.3.18.jar:5.3.18]
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:117) ~[spring-web-5.3.18.jar:5.3.18]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189) ~[tomcat-embed-core-9.0.60.jar:9.0.60]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162) ~[tomcat-embed-core-9.0.60.jar:9.0.60]
	at org.springframework.web.filter.CharacterEncodingFilter.doFilterInternal(CharacterEncodingFilter.java:201) ~[spring-web-5.3.18.jar:5.3.18]
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:117) ~[spring-web-5.3.18.jar:5.3.18]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189) ~[tomcat-embed-core-9.0.60.jar:9.0.60]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162) ~[tomcat-embed-core-9.0.60.jar:9.0.60]
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:197) ~[tomcat-embed-core-9.0.60.jar:9.0.60]
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97) ~[tomcat-embed-core-9.0.60.jar:9.0.60]
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:541) ~[tomcat-embed-core-9.0.60.jar:9.0.60]
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:135) ~[tomcat-embed-core-9.0.60.jar:9.0.60]
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92) ~[tomcat-embed-core-9.0.60.jar:9.0.60]
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78) ~[tomcat-embed-core-9.0.60.jar:9.0.60]
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:360) ~[tomcat-embed-core-9.0.60.jar:9.0.60]
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:399) ~[tomcat-embed-core-9.0.60.jar:9.0.60]
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65) ~[tomcat-embed-core-9.0.60.jar:9.0.60]
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:889) ~[tomcat-embed-core-9.0.60.jar:9.0.60]
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1743) ~[tomcat-embed-core-9.0.60.jar:9.0.60]
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49) ~[tomcat-embed-core-9.0.60.jar:9.0.60]
	at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1191) ~[tomcat-embed-core-9.0.60.jar:9.0.60]
	at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659) ~[tomcat-embed-core-9.0.60.jar:9.0.60]
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61) ~[tomcat-embed-core-9.0.60.jar:9.0.60]
	at java.base/java.lang.Thread.run(Thread.java:833) ~[na:na]
Caused by: com.vaadin.flow.server.ExecutionFailedException: Hilla Generator execution failed with exit code 1
	at dev.hilla.internal.TaskGenerateHillaImpl.runCodeGeneration(TaskGenerateHillaImpl.java:54) ~[engine-runtime-1.2.0.rc1.jar:na]
	at dev.hilla.internal.TaskGenerateHillaImpl.execute(TaskGenerateHillaImpl.java:39) ~[engine-runtime-1.2.0.rc1.jar:na]
	at com.vaadin.flow.server.frontend.NodeTasks.execute(NodeTasks.java:959) ~[flow-server-23.2.0.rc2.jar:23.2.0.rc2]
	at com.vaadin.base.devserver.startup.DevModeInitializer.runNodeTasks(DevModeInitializer.java:434) ~[vaadin-dev-server-23.2.0.rc2.jar:na]
	at com.vaadin.base.devserver.startup.DevModeInitializer.lambda$initDevModeHandler$0(DevModeInitializer.java:349) ~[vaadin-dev-server-23.2.0.rc2.jar:na]
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.run$$$capture(CompletableFuture.java:1804) ~[na:na]
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java) ~[na:na]
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1796) ~[na:na]
	at java.base/java.util.concurrent.ForkJoinTask.doExec$$$capture(ForkJoinTask.java:373) ~[na:na]
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java) ~[na:na]
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182) ~[na:na]
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655) ~[na:na]
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622) ~[na:na]
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165) ~[na:na]


### Expected behavior

That it should compile code

### Minimal reproducible example

Don't have

### Versions

Hilla: 23.2.0.rc1
Flow: 23.2.0.rc2
Java: JetBrains s.r.o. 17.0.2
OS: amd64 Windows 10 10.0
Browser: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/104.0.0.0 Safari/537.36
Live reload: Java active (HotswapAgent): Front end active

--- a/packages/java/parser-jvm-plugin-backbone/src/main/java/dev/hilla/parser/plugins/backbone/SchemaProcessor.java
+++ b/packages/java/parser-jvm-plugin-backbone/src/main/java/dev/hilla/parser/plugins/backbone/SchemaProcessor.java
@@ -137,13 +137,17 @@ final class SchemaProcessor {
     }
 
     private Schema<?> mapSchema() {
+        var schema = nullify(new MapSchema(), true);
+
         var _type = (ClassRefSignatureModel) type;
         var typeArguments = _type.getTypeArguments();
-        var values = new SchemaProcessor(typeArguments.get(1), info, storage)
-                .process();
 
-        var schema = nullify(new MapSchema(), true)
-                .additionalProperties(values);
+        // For the TS generator, to recognize a schema as a map, it requires
+        // "additionalProperties" to be set
+        schema.additionalProperties(!typeArguments.isEmpty()
+                ? new SchemaProcessor(typeArguments.get(1), info, storage)
+                        .process()
+                : anySchema());
 
         if (type.isNonJDKClass()) {
             schema.addExtension("x-class-name", _type.getName());
@@ -161,8 +165,10 @@ final class SchemaProcessor {
         var _type = (ClassRefSignatureModel) type;
         var typeArguments = _type.getTypeArguments();
 
-        return new SchemaProcessor(typeArguments.get(0), info, storage)
-                .process();
+        return !typeArguments.isEmpty()
+                ? new SchemaProcessor(typeArguments.get(0), info, storage)
+                        .process()
+                : anySchema();
     }
 
     private Schema<?> refSchema() {
@@ -185,8 +191,9 @@ final class SchemaProcessor {
     private Schema<?> typeArgumentSchema() {
         var types = ((TypeArgumentModel) type).getAssociatedTypes();
 
-        return types.isEmpty() ? anySchema()
-                : new SchemaProcessor(types.get(0), info, storage).process();
+        return !types.isEmpty()
+                ? new SchemaProcessor(types.get(0), info, storage).process()
+                : anySchema();
     }
 
     private Schema<?> typeParameterSchema() {

--- a/packages/java/parser-jvm-plugin-backbone/src/test/java/dev/hilla/parser/plugins/backbone/bare/BareTypeEndpoint.java
+++ b/packages/java/parser-jvm-plugin-backbone/src/test/java/dev/hilla/parser/plugins/backbone/bare/BareTypeEndpoint.java
@@ -1,0 +1,20 @@
+package dev.hilla.parser.plugins.backbone.bare;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+@Endpoint
+public class BareTypeEndpoint {
+    public Map getBareMap() {
+        return null;
+    }
+
+    public List getBareList() {
+        return null;
+    }
+
+    public Optional getBareOptional() {
+        return Optional.empty();
+    }
+}

--- a/packages/java/parser-jvm-plugin-backbone/src/test/java/dev/hilla/parser/plugins/backbone/bare/BareTypeTest.java
+++ b/packages/java/parser-jvm-plugin-backbone/src/test/java/dev/hilla/parser/plugins/backbone/bare/BareTypeTest.java
@@ -1,0 +1,26 @@
+package dev.hilla.parser.plugins.backbone.bare;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.util.Set;
+
+import org.junit.jupiter.api.Test;
+
+import dev.hilla.parser.core.ParserConfig;
+import dev.hilla.parser.plugins.backbone.BackbonePlugin;
+import dev.hilla.parser.plugins.backbone.test.helpers.TestHelper;
+
+public class BareTypeTest {
+    private final TestHelper helper = new TestHelper(getClass());
+
+    @Test
+    public void should_CorrectlyResolveBareTypes()
+            throws IOException, URISyntaxException {
+        var config = new ParserConfig.Builder()
+                .classPath(Set.of(helper.getTargetDir().toString()))
+                .endpointAnnotation(Endpoint.class.getName())
+                .addPlugin(new BackbonePlugin()).finish();
+
+        helper.executeParserWithConfig(config);
+    }
+}

--- a/packages/java/parser-jvm-plugin-backbone/src/test/java/dev/hilla/parser/plugins/backbone/bare/Endpoint.java
+++ b/packages/java/parser-jvm-plugin-backbone/src/test/java/dev/hilla/parser/plugins/backbone/bare/Endpoint.java
@@ -1,0 +1,11 @@
+package dev.hilla.parser.plugins.backbone.bare;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.TYPE)
+public @interface Endpoint {
+}

--- a/packages/java/parser-jvm-plugin-backbone/src/test/java/dev/hilla/parser/plugins/backbone/wildcard/Endpoint.java
+++ b/packages/java/parser-jvm-plugin-backbone/src/test/java/dev/hilla/parser/plugins/backbone/wildcard/Endpoint.java
@@ -1,0 +1,11 @@
+package dev.hilla.parser.plugins.backbone.wildcard;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.TYPE)
+public @interface Endpoint {
+}

--- a/packages/java/parser-jvm-plugin-backbone/src/test/java/dev/hilla/parser/plugins/backbone/wildcard/WildcardTypeEndpoint.java
+++ b/packages/java/parser-jvm-plugin-backbone/src/test/java/dev/hilla/parser/plugins/backbone/wildcard/WildcardTypeEndpoint.java
@@ -1,0 +1,20 @@
+package dev.hilla.parser.plugins.backbone.wildcard;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+@Endpoint
+public class WildcardTypeEndpoint {
+    public Map<String, ?> getDefaultWildcard() {
+        return null;
+    }
+
+    public List<? extends Map<Object, Object>> getExtendingWildcard() {
+        return null;
+    }
+
+    public Optional<? super List<Object>> getSuperWildcard() {
+        return Optional.empty();
+    }
+}

--- a/packages/java/parser-jvm-plugin-backbone/src/test/java/dev/hilla/parser/plugins/backbone/wildcard/WildcardTypeTest.java
+++ b/packages/java/parser-jvm-plugin-backbone/src/test/java/dev/hilla/parser/plugins/backbone/wildcard/WildcardTypeTest.java
@@ -1,0 +1,26 @@
+package dev.hilla.parser.plugins.backbone.wildcard;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.util.Set;
+
+import org.junit.jupiter.api.Test;
+
+import dev.hilla.parser.core.ParserConfig;
+import dev.hilla.parser.plugins.backbone.BackbonePlugin;
+import dev.hilla.parser.plugins.backbone.test.helpers.TestHelper;
+
+public class WildcardTypeTest {
+    private final TestHelper helper = new TestHelper(getClass());
+
+    @Test
+    public void should_CorrectlyResolveWildcardTypes()
+            throws IOException, URISyntaxException {
+        var config = new ParserConfig.Builder()
+                .classPath(Set.of(helper.getTargetDir().toString()))
+                .endpointAnnotation(Endpoint.class.getName())
+                .addPlugin(new BackbonePlugin()).finish();
+
+        helper.executeParserWithConfig(config);
+    }
+}

--- a/packages/java/parser-jvm-plugin-backbone/src/test/resources/dev/hilla/parser/plugins/backbone/bare/openapi.json
+++ b/packages/java/parser-jvm-plugin-backbone/src/test/resources/dev/hilla/parser/plugins/backbone/bare/openapi.json
@@ -1,0 +1,80 @@
+{
+  "openapi": "3.0.1",
+  "info": {
+    "title": "Vaadin Application",
+    "version": "1.0.0"
+  },
+  "servers": [
+    {
+      "url": "http://localhost:8080/connect",
+      "description": "Vaadin Backend"
+    }
+  ],
+  "tags": [
+    {
+      "name": "BareTypeEndpoint"
+    }
+  ],
+  "paths": {
+    "/BareTypeEndpoint/getBareMap": {
+      "post": {
+        "tags": ["BareTypeEndpoint"],
+        "operationId": "BareTypeEndpoint_getBareMap_POST",
+        "responses": {
+          "200": {
+            "description": "",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "additionalProperties": {
+                    "type": "object"
+                  },
+                  "nullable": true
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/BareTypeEndpoint/getBareList": {
+      "post": {
+        "tags": ["BareTypeEndpoint"],
+        "operationId": "BareTypeEndpoint_getBareList_POST",
+        "responses": {
+          "200": {
+            "description": "",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "array",
+                  "nullable": true
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/BareTypeEndpoint/getBareOptional": {
+      "post": {
+        "tags": ["BareTypeEndpoint"],
+        "operationId": "BareTypeEndpoint_getBareOptional_POST",
+        "responses": {
+          "200": {
+            "description": "",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object"
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  },
+  "components": {}
+}

--- a/packages/java/parser-jvm-plugin-backbone/src/test/resources/dev/hilla/parser/plugins/backbone/wildcard/openapi.json
+++ b/packages/java/parser-jvm-plugin-backbone/src/test/resources/dev/hilla/parser/plugins/backbone/wildcard/openapi.json
@@ -1,0 +1,91 @@
+{
+  "openapi": "3.0.1",
+  "info": {
+    "title": "Vaadin Application",
+    "version": "1.0.0"
+  },
+  "servers": [
+    {
+      "url": "http://localhost:8080/connect",
+      "description": "Vaadin Backend"
+    }
+  ],
+  "tags": [
+    {
+      "name": "WildcardTypeEndpoint"
+    }
+  ],
+  "paths": {
+    "/WildcardTypeEndpoint/getDefaultWildcard": {
+      "post": {
+        "tags": ["WildcardTypeEndpoint"],
+        "operationId": "WildcardTypeEndpoint_getDefaultWildcard_POST",
+        "responses": {
+          "200": {
+            "description": "",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "additionalProperties": {
+                    "type": "object"
+                  },
+                  "nullable": true
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/WildcardTypeEndpoint/getExtendingWildcard": {
+      "post": {
+        "tags": ["WildcardTypeEndpoint"],
+        "operationId": "WildcardTypeEndpoint_getExtendingWildcard_POST",
+        "responses": {
+          "200": {
+            "description": "",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "array",
+                  "nullable": true,
+                  "items": {
+                    "type": "object",
+                    "additionalProperties": {
+                      "type": "object"
+                    },
+                    "nullable": true
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/WildcardTypeEndpoint/getSuperWildcard": {
+      "post": {
+        "tags": ["WildcardTypeEndpoint"],
+        "operationId": "WildcardTypeEndpoint_getSuperWildcard_POST",
+        "responses": {
+          "200": {
+            "description": "",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "array",
+                  "nullable": true,
+                  "items": {
+                    "type": "object"
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  },
+  "components": {}
+}

